subsys/partition_manager/Kconfig

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menuconfig PM_EXTERNAL_FLASH
+	bool "Support external flash in Partition Manager"
+
+if PM_EXTERNAL_FLASH
+
+config PM_EXTERNAL_FLASH_DEV_NAME
+	string "External flash dev name"
```

Comment:
```suggestion
	string "Name of the external flash device"
```

Reply:
Updated, also added help text, please check this.